airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/message/Batch.kt

```diff
@@ -60,6 +63,8 @@ interface Batch {
 data class SimpleBatch(override val state: Batch.State) : Batch
 
 /** Represents a file of records locally staged. */
+// In Kotlin, abstract classes really are only around for java backwards compatibility.
+// Instead you want either an interface or a sealed class here.
 abstract class StagedLocalFile() : Batch {
```

Comment:
I actually thought I'd made this an interface, because the point is to mix it in as needed. (See previous comment about `Batch`.)